src/main/java/org/myrobotlab/service/InMoovTorso.java

```diff
@@ -264,10 +268,12 @@ public void setpins(Integer topStomPin, Integer midStomPin, Integer lowStomPin)
     this.lowStom.setPin(lowStom);
     */
 	  
+    /** FIXME - has to be done outside of

```

Comment:
not backwards compatible...   should be refactored to set the pin on the servo and attach it to the arduino.

Reply:
fixed